tests/system/small/operations/test_semantics.py

```diff
@@ -62,3 +62,53 @@ def test_filter_invalid_instruction_raise_error(instruction, gemini_flash_model)
 
     with pytest.raises(ValueError):
         df.semantics.filter(instruction, gemini_flash_model)
+
+
+def test_map(session, gemini_flash_model):
+    bigframes.options.experiments.semantic_operators = True
+    df = dataframe.DataFrame(
+        data={
+            "ingredient_1": ["Burger Bun", "Soy Bean"],
+            "ingredient_2": ["Beef Patty", "Bittern"],
+        },
+        session=session,
+    )
+
+    actual_df = df.semantics.map(
+        "What is the food made from {ingredient_1} and {ingredient_2}? One word only.",
+        gemini_flash_model,
+    ).to_pandas()
+    # Result sanitation
+    actual_df["map_result"] = actual_df["map_result"].str.strip().str.lower()
+
+    expected_df = pd.DataFrame(
+        {
+            "ingredient_1": ["Burger Bun", "Soy Bean"],
+            "ingredient_2": ["Beef Patty", "Bittern"],
+            "map_result": ["burger", "tofu"],
```

Comment:
Can LM return the same results for this questions? Maybe just ask `Which city is the capital of {country}`?

Reply:
For now LM responses are pretty stable for obvious questions. I think we should be fine here.